src/sd_jwt/bin/generate.py

```diff
@@ -42,12 +57,22 @@ def generate_test_case_data(settings: Dict, testcase_path: Path, type: str):
     extra_header_parameters = testcase.get("extra_header_parameters", {})
 
     claims = {}
+
+    _iat = get_value_from_disclosables("iat", testcase)
+
+    iat = _iat or settings.get("iat", int(datetime.datetime.utcnow().timestamp()))
+    exp = iat + (settings.get("exp_delta_seconds", 60) * 60)
+    
     if include_default_claims:
         claims = {
             "iss": settings["identifiers"]["issuer"],
-            "iat": settings["iat"],
-            "exp": settings["exp"],
+            "exp": settings.get("exp", exp)
         }
+    else:
+        claims = dict()
+
+    if not _iat:
+        claims['iat'] = iat
```

Comment:
this must be refactored

I can remove that `if` and use `claims['iat'] = _iat or iat`